setup_env.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you add `cd $(dirname "$0")/..` at the start to ensure this is always running with project root as working directory?